app/api/v2/resource/api_keys.rb

```diff
@@ -33,7 +33,7 @@ class APIKeys < Grape::API
           optional :scope,
```

Comment:
@ec you can specify this parameter type as JSON
https://github.com/ruby-grape/grape#first-class-json-types 
This should give an ability to validate scopes on fly